src/RequestBuilder/DocumentValidationRequestBuilder.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+
+namespace AppBundle\GatewaySDKPhp\RequestBuilder;
+
+
+use AppBundle\GatewaySDKPhp\Exception\MissingParameterException;
+use AppBundle\GatewaySDKPhp\Model\Request;
+use AppBundle\GatewaySDKPhp\Model\RequestInterface;
+use AppBundle\GatewaySDKPhp\RequestBuilder\Partials\FileUpload;
+use AppBundle\GatewaySDKPhp\RequestBuilder\Partials\Signer;
+use AppBundle\GatewaySDKPhp\RequestBuilder\Traits\TraitBuildParameters;
+use Symfony\Component\Validator\Validation;
+use Symfony\Component\Validator\Constraints AS Assert;
+
+class DocumentValidationRequestBuilder extends AbstractRequestBuilder
+{
+    use TraitBuildParameters;
+
+    /**
+     * @var string
+     */
+    protected $access_token;
```

Comment:
$accessToken would be better in that case

Reply:
Actually, I'm building the parameters dynamically from class attributes, and as the parameter is named as "access_token" in the APIs, I have to keep it in that way.